### PR TITLE
docs: refine the tutorial for lime and add tutorial for attributions methods

### PR DESCRIPTION
**Pull request by WolodjaZ**

### CODE REVIEW

File: examples/hyperview_challenge/attr_showcase.ipynb
Comment (Fersoil) [inline examples/hyperview_challenge/attr_showcase.ipynb:1]: In general, good job. Apart from Przemek's comments, I would add:
- a little bit shorter title
- I like a bit more the intro in the LIME tutorial - I would make it consistent accross the tutorials
- better explanation of HyperNoiseTunnel - this is extra created by us, so I would dwell into it a little bit more
- The tutorial lacks explanations of the figures - for the LIME one, you proposed a very nice solution to describe the chart contents. Here I would do it at least once
- consider moving the function in the first section to a separate file - the user won't read the code anyway and it clutters the tutorial
- consider adding a better description of a parameters in an `attribute` method call

File: examples/hyperview_challenge/lime.ipynb
Comment (Fersoil) [inline examples/hyperview_challenge/lime.ipynb:1]: Looks good. Again Przemek is right about reducing the block of wavelength listing. From my perspective you could also move the model download into a separate file

**Review by Fersoil — CHANGES_REQUESTED** [review]
There are needed some changes pointed by Przemek, I would also add more detailed descriptions for the attribute methods showcase

**Comment by WolodjaZ** [comment]
@Fersoil check if the file `wavelengths` in segmentation data example is correct :)

**Review by Fersoil — APPROVED** [review]
only one small issue with attr tutorial, but good to merge

**Pull request merged**